parm/config/gefs/yaml/defaults.yaml

```diff
@@ -9,7 +9,7 @@ base:
   DO_AWIPS: "NO"
   KEEPDATA: "NO"
   DO_EXTRACTVARS: "NO"
-  FHMAX_GFS: 120
+  FCST_SEGMENTS_GFS: "0,48,120"
```

Comment:
Just for my own understanding, does this mean run a forecast from f000 through f048, then another from f048 through f120?

Reply:
Yes.